src/test/java/org/skriptlang/skript/test/tests/syntaxes/InventoryMoveItemEventTest.java

```diff
@@ -0,0 +1,45 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package org.skriptlang.skript.test.tests.syntaxes;
+
+import ch.njol.skript.test.runner.SkriptJUnitTest;
+import org.bukkit.Bukkit;
+import org.bukkit.Material;
+import org.bukkit.event.inventory.InventoryMoveItemEvent;
+import org.bukkit.event.inventory.InventoryType;
+import org.bukkit.inventory.Inventory;
+import org.bukkit.inventory.ItemStack;
+import org.junit.Test;
+
+// TODO properly add test after merge of https://github.com/SkriptLang/Skript/pull/6261
```

Comment:
This has been merged now btw

Reply:
Can we add the new test in 2.8.1? I'd like to the fix itself in for 2.8.0.

Reply:
I’m still facing issues with JUnit despite the merge of the mentioned PR, still awaiting for assistance in skriptlang-development channel.

Reply:
#6276 has to be merged first before I can fix this tests.

Reply:
Sounds good. I also forgot that we can do multiple pre-releases, so it's fine to have this added for pre-release 2